source/runner/_tests/fixtures/__DangerfilePlugin.js

```diff
@@ -0,0 +1,7 @@
+import { checkForTypesInDeps } from "danger-plugin-yarn"
+const deps = {
+  dependencies: {
+    added: ["@types/danger"],
+  },
+}
+checkForTypesInDeps(deps)
```

Comment:
this is a weird-ish test, as it's making assertions about how `danger-plugin-yarn` works, but I'd rather have a test for something, and this seems reasonable enough